src/Neo/UInt160.cs

```diff
@@ -14,6 +14,8 @@
 using System;
 using System.Globalization;
 using System.IO;
+using System.Linq;
```

Comment:
```suggestion
```
It's used?

Reply:
yes, it is being used  var data = Enumerable.Range(0, Length)